backend/internal/service/handler/review.go

```diff
@@ -0,0 +1,104 @@
+package handler
+
+import (
+	"fmt"
+	"platnm/internal/storage"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type ReviewHandler struct {
+	reviewRepository storage.ReviewRepository
+}
+
+func NewReviewHandler(reviewRepository storage.ReviewRepository) *ReviewHandler {
+	return &ReviewHandler{
+		reviewRepository,
+	}
+}
+
+func (h *ReviewHandler) GetReviews(c *fiber.Ctx) error {
+	review, err := h.reviewRepository.GetReviews(c.Context())
+	if err != nil {
+		return err
+	}
+
+	return c.Status(fiber.StatusOK).JSON(review)
+}
+
+func (h *ReviewHandler) GetReviewById(c *fiber.Ctx, mediaType string) error {
```

Comment:
not sure if having one handler here and dynamically switching between media type is necessarily bad, but maybe we want to follow one handler per route

but not sure tbh

Reply:
nice work! just some minor suggestions